phoonnx_train/optispeech/model/generator/modules/convnext.py

```
@@ -0,0 +1,131 @@
+    """
+    ConvNeXt Block adapted from https://github.com/facebookresearch/ConvNeXt to 1D audio signal.
+
+    Args:
+        dim (int): Number of input channels.
+        intermediate_dim (int): Dimensionality of the intermediate layer.
+        drop_path (float): Stochastic depth rate. Default: 0.0
+        layer_scale_init_value (float, optional): Initial value for the layer scale. None means no scaling.
+            Defaults to None.
+    """
+
+    def __init__(self, dim: int, intermediate_dim: int, drop_path: float = 0.0, layer_scale_init_value: float = None):
```

Ruff (0.15.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)